book/_toc.yml

```diff
@@ -5,7 +5,6 @@ format: jb-book
 parts:
 - caption: Details
   chapters:
-    - file: logistics
```

Comment:
Would it be worth-while changing this to a link to the Notion logistics page, rather than removing it entirely?

Reply:
Good idea! I don't have strong preference (and I think that's a great idea for the template - I opened an issue to track ideas for future handling of logistics info over in the [template repo](https://github.com/uwhackweek/jupyterbook-template/issues/183). For the present, I'll note we've removed it from the other two books prior to my seeing this message.